### DISCUSSION TITLE
getFiberRobots

### DISCUSSION THREAD

**Discussion by mtahagocer** [vercel/next.js#18110] [Help]
This probobly about active devtools.
backend.bundle.js:1 Uncaught TypeError: Cannot read property 'getFiberRoots' of undefined
at backend.bundle.js:1
at Module. (backend.bundle.js:1)
at r (backend.bundle.js:1)
at backend.bundle.js:1
at backend.bundle.js:1

**Comment by jamesmosier** [comment]
Hi @mtahagocer. Do you have a repository you could share?
